src/Functions/keyvaluepair/tests/gtest_extractKeyValuePairs.cpp

```diff
@@ -82,9 +64,7 @@ TEST_P(extractKVPairKeyValuePairExtractorTest, Match)
     auto values = ColumnString::create();
 
     auto pairs_found = kv_parser->extract(input, keys, values);
-    ASSERT_EQ(expected.size(), pairs_found)
-            << "\texpected: " << print_with_dump(expected) << "\n"
-            << "\tactual  : " << print_with_dump(*ToColumnMap(keys, values));
+    ASSERT_EQ(expected.size(), pairs_found);
```

Comment:
`<<` would print extra info when a check fails which is really useful for debugging test failures.
I see it's only in a few places but I would still be careful with removing messages like this.

Reply:
We don't appreciate unit tests for SQL functions. Most likely, this test was added by mistake.